Packs/NetskopeV2/Integrations/NetskopeV2/README.md

```diff
@@ -10,11 +10,13 @@ This integration was integrated and tested with version 91.0.6.575 of Netskope (
     | **Parameter** | **Required** |
     | --- | --- |
     | URL of Netskope Tenant (e.g. https://tenant.goskope.com) | True |
-    | API Key | True |
+    | API Key | False |
+    | API Key | False |
```

Comment:
why did you add it twice?

Reply:
I didn't, I don't believe I have control of this when contributing directly from the XSOAR UI